src/main/java/fr/insee/queen/api/configuration/KeycloakConfig.java

```diff
@@ -28,13 +29,17 @@
 @KeycloakConfiguration
 public class KeycloakConfig extends KeycloakWebSecurityConfigurerAdapter {
 
+	@Value("${fr.insee.queen.interviewer.role:#{null}}")
```

Comment:
Should be added to readme with definition of all properties read in external property file

Reply:
Done, Readme is up to date